source/test/TestApp/PlayFabApiTest.cpp

```diff
@@ -730,6 +767,43 @@ namespace PlayFabUnit
         }
     }
 
+     // GROUPS API
+    void PlayFabApiTest::GroupsApiTest(TestContext& testContext)
+    {
+        auto settings = std::make_shared<PlayFabApiSettings>();
+        //clientApi = std::make_shared<PlayFab::PlayFabClientInstanceAPI>(settings);
```

Comment:
Delete unused code comments.
Also, this is definitely something that should only exist in a setup step, not in the middle of a test.